src/libs/elektra/kdb.c

```diff
@@ -11,6 +11,10 @@
 #include "kdbconfig.h"
 #endif
 
+#if DEBUG && defined(HAVE_STDIO_H)
+#include <stdio.h>
+#endif
+
```

Comment:
probably something because the backend branch is so old?

please remove and use logger

Reply:
Yes, all these weird changes (especially in `kdb.c`) are most likely because of mistakes during the last merge from `master`.